azure/README.md

```diff
@@ -0,0 +1,104 @@
+# Pytorch Elastic on Azure
```

Comment:
Hello @aashna! Thank you for this contribution! I've added a few comments below.